src/main/java/com/google/sps/servlets/userServlet.java

```diff
@@ -10,13 +19,62 @@
 @WebServlet("/userinformation")
 public class userServlet extends HttpServlet {
```

Comment:
It'll be good to follow the naming convention for Java classes (first letter of each word capitalized, i.e. UserServlet)!